src/bindings/python/src/openvino/__init__.py

```diff
@@ -21,6 +21,7 @@
 from openvino import runtime as runtime
 from openvino import frontend as frontend
 from openvino import helpers as helpers
+from openvino import dev_api as dev_api
```

Comment:
since it's dev API and required for advanced users, do we need to expose it in root init py?